engine/apps/alerts/migrations/0066_remove_channelfilter__slack_channel_id_and_more.py

```diff
@@ -11,15 +10,6 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        linter.IgnoreMigration(),
-        migrations.RemoveField(
-            model_name='channelfilter',
-            name='_slack_channel_id',
-        ),
-        migrations.RemoveField(
-            model_name='resolutionnoteslackmessage',
-            name='_slack_channel_id',
-        ),
```

Comment:
moved to `engine/apps/alerts/migrations/0068_remove_resolutionnoteslackmessage__slack_channel_id_state.py`